Dockerfile-ngc

```diff
@@ -0,0 +1,201 @@
+ARG BASE_IMAGE
+FROM ${BASE_IMAGE}
+
+RUN mkdir -p /var/run/sshd

```

Comment:
I thought this block was moved to a new script to hide the apt-get installs. Is this up to date with the latest ngc_images branch? May not be necessary for this commit.

Reply:
The latest ngc_images branch does not have "Dockerfile-ngc".  They have "Dockerfile-ngc-hpc".  The "Dockerfile-ngc" is used by our target "build-gpu-ngc-gh-base"

Reply:
Ok is the plan to update the dockerfile used after this is merged and we rebase with the latest?